### DISCUSSION TITLE
How about a pip-installable distribution?

### DISCUSSION THREAD

**Discussion by jaraco** [brettcannon/python-launcher#390] [Ideas]
I'd probably not have suggested this prior to seeing the example in ruff/uv.

How about a pip installable distribution? Basically, it would be a series of wheels, probably built by cibuildwheel, that when installed expose the `py` command in the bin/scripts directory. This would allow people without a suitable platform packager (or one without a recipe for python-launcher) to `pipx install python-launcher` (or similar) to get the command.

**Comment by brettcannon** [comment]
I have toyed w/ the idea. The key question is whether the increased release burden is worth it.
